README.md

```diff
@@ -1,2 +1,193 @@
-# hello_microverse
-Day 2 Microverse Program on github flow, use of linters and professional readme.md 
+<a name="readme-top"></a>
+<div align="center">
+  <img src="jojo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  # Hello Microverse  Project  Documentation
+</div>
```

Comment:
Kindly remove these lines of codes as it is not an essential p[part of the README file 😄

Reply:
Sorted out